src/Component/Image/index.js

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import styled from "styled-components";
+
+const StyledImage = styled.img`
+  position: relative;
+  object-fit: cover;
+  width: ${(props) => (props.width ? props.width : "16px")} ;
+  height: ${(props) => (props.height ? props.height : "16px")} ;
+`;
+
+const Image = (props) => {
+  return (
+    <StyledImage src={props.Image} width={props.width} height={props.height}>
+      {props.children}
+    </StyledImage>
+  );
+};
+
+export default Image;
```

Comment:
여기서 `Image` 컴포넌트는 단순히 props 를 받아 그대로 `StyledImage` 에게 전달하는 역할만을 하고 있는데요,

```suggestion
const Image = styled.img`
  position: relative;
  object-fit: cover;
  width: ${(props) => (props.width ? props.width : "16px")} ;
  height: ${(props) => (props.height ? props.height : "16px")} ;
`;

export default Image;
```

위와 같이 코드를 개선하면 더 가독성이 높아질 수 있겠군요 🧐

Reply:
넵! 참고하겠습니당:)